easybuild/easyblocks/v/vtune.py

```diff
@@ -73,25 +74,26 @@ def make_module_req_guess(self):
         """
 
         guesses = super(EB_VTune, self).make_module_req_guess()
+        # List of environment variables that the module shouldn't set
+        do_not_set = ['CPATH', 'LD_LIBRARY_PATH', 'LIBRARY_PATH']
 
         if self.cfg['m32']:
             guesses.update({
                 'PATH': [os.path.join(self.subdir, 'bin32')],
-                'LD_LIBRARY_PATH': [os.path.join(self.subdir, 'lib32')],
-                'LIBRARY_PATH': [os.path.join(self.subdir, 'lib32')],
             })
         else:
             guesses.update({
                 'PATH': [os.path.join(self.subdir, 'bin64')],
-                'LD_LIBRARY_PATH': [os.path.join(self.subdir, 'lib64')],
-                'LIBRARY_PATH': [os.path.join(self.subdir, 'lib64')],
             })
 
         guesses.update({
-            'CPATH': [os.path.join(self.subdir, 'include')],
             'MANPATH': [os.path.join(self.subdir, 'man')],
         })
 
+        for ev in do_not_set:
```

Comment:
`do_not_set` is not used anywhere else, so no need to introduce a variable

also, clarify with a comment why this is being done, and include (debug) log messages for kicking those out:

```python
# make sure $CPATH, $LD_LIBRARY_PATH and $LIBRARY_PATH are not updated in generated module file,
# because that leads to problem when the libraries included with VTune are being picked up
for key in ['CPATH', 'LD_LIBRARY_PATH', 'LIBRARY_PATH']:
    if key in guesses:
        self.log.debug("Purposely ot updating $%s in VTune module file", key)
        del guesses[key]

Reply:
@damianam ping?

Reply:
@boegel pong back at ya'!